### ISSUE TITLE
Cant configure

### ISSUE THREAD

**Issue by xaviondk** [Dielee/volvo2mqtt#90]
**Describe the bug**
Getting an error message when trying to save my configuration
![Screenshot_20230822_141852_Home Assistant](https://github.com/Dielee/volvo2mqtt/assets/68909910/338d24c9-da37-4c9d-ae88-522692b6a226)

**To Reproduce**:<br>

**Expected behavior**:<br>

**Screenshots**:<br>

**Version info**:<br>
The version you are using and the method you are using (Docker container/HA Addon/Cloned repo)

**Logs**:<br>
Enable the debug option and upload your logfile here. <br>
<b>If you are using the HA Addon the log location is `\\<Your HA Host IP>\addons\volvo2mqtt\log\volvo2mqtt.log`

**Comment by Dielee** [comment]
You messed up something with your config. The whole "volvoData" section is missing. Please reinstall and retry.

**Comment by Dielee** [comment]
If not, please share me you whole config page as screenshot!

**Comment by xaviondk** [comment]
> If not, please share me you whole config page as screenshot!

Getting same error after a reinstall. 
![Screenshot_20230822_144504_Home Assistant~2](https://github.com/Dielee/volvo2mqtt/assets/68909910/3c50eada-8261-4d2c-a94b-9cff54213805)

**Comment by Dielee** [comment]
Your vcc api key, have to be a list. Like this: 

  ```
    vccapikey:
      - vccapikey1
      - vccapikey2
      - vccapikey3
      - etc.
  ```

Pls take a look at the changelog!

**Comment by xaviondk** [comment]
Changed it to list form, but still getting the same error with volvodata missing

**Comment by Dielee** [comment]
Please try to configure on a pc... maybe something is messing up. I have no other idea, sorry.

**Comment by TrekBonanza** [comment]
Had same problem with same error message.
After I updated to a version with the opportunity to multi keys my existing key was still one the same row as vccapikey: xxxxxxxxxxxxxxxxxxx and it generate the error and it stopped working.
I moved down the key a row wit a - to make a list and then it was possible to save without error.

Maybe move down the "" a row with - to make a list direcly and avoid misunderstandning?
  volvoData:
    username: null
    password: null
    vin: ""
    vccapikey: null
    odometerMultiplier: 1
    averageSpeedDivider: 1
    averageFuelConsumptionMultiplier: 1
